src/components/MenuItem.tsx

```diff
@@ -0,0 +1,599 @@
+            processedTitle = html;
+        }
+
+        return processedTitle ? `<comment>${processedTitle}</comment>` : '';
+    }, [title, shouldRenderAsHTML, shouldParseTitle, html]);
+
+    const hasPressableRightComponent = iconRight || (shouldShowRightComponent && rightComponent);
+
+    const renderTitleContent = () => {
+        if (title && titleWithTooltips && Array.isArray(titleWithTooltips) && titleWithTooltips.length > 0) {
+            return (
+                <DisplayNames
+                    fullTitle={title}
+                    displayNamesWithTooltips={titleWithTooltips}
+                    tooltipEnabled
+                    numberOfLines={1}
+                />
+            );
+        }
+
+        return title ? convertToLTR(title) : '';
+    };
+
+    const onPressAction = (event: GestureResponderEvent | KeyboardEvent | undefined) => {
+        if (disabled || !interactive) {
+            return;
+        }
+
+        if (event && event.type === 'click') {
+            (event.currentTarget as HTMLElement).blur();
+        }
+
+        if (onPress && event) {
+            onPress(event);
+        }
+    };
+
+    return (
+        <Hoverable>
+            {(isHovered) => (
+                <PressableWithSecondaryInteraction
+                    onPress={shouldCheckActionAllowedOnPress ? Session.checkIfActionIsAllowed((args) => onPressAction(args as GestureResponderEvent), isAnonymousAction) : onPressAction}
+                    onPressIn={() => shouldBlockSelection && isSmallScreenWidth && DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
+                    onPressOut={ControlSelection.unblock}
+                    onSecondaryInteraction={onSecondaryInteraction}
+                    style={({pressed}: PressableStateCallbackType) =>
+                        [
+                            containerStyle,
+                            errorText ? styles.pb5 : {},
+                            combinedStyle,
+                            !interactive && styles.cursorDefault,
+                            StyleUtils.getButtonBackgroundColorStyle(getButtonState(focused || isHovered, Boolean(pressed), success, disabled, interactive), true),
+                            (isHovered || pressed) && hoverAndPressStyle,
+                            ...(Array.isArray(wrapperStyle) ? wrapperStyle : [wrapperStyle]),
+                            shouldGreyOutWhenDisabled && disabled && styles.buttonOpacityDisabled,
+                        ] as StyleProp<ViewStyle>
+                    }
+                    disabled={disabled}
+                    ref={ref}
+                    role={CONST.ROLE.MENUITEM}
+                    accessibilityLabel={title ? title.toString() : ''}
+                    accessible
+                >
+                    {({pressed}) => (
+                        <>
+                            <View style={[styles.flexColumn, styles.flex1]}>
+                                {Boolean(label) && (
```

Comment:
Change every Boolean(...) to !!(...)

Reply:
OK. Doing it for everything except for booleans - for those just simply removing Boolean().